compiler/codegen/OMRCodeGenerator.hpp

```diff
@@ -371,6 +371,7 @@ class OMR_EXTENSIBLE CodeGenerator
    TR_HasRandomGenerator randomizer;
 
    virtual bool supportsAtomicAdd() {return false;}
+   virtual bool hasTMEvaluator()    {return false;}
```

Comment:
Daryl's earlier comment that these don't need to be virtual is right because CodeGenerator is an extensible class. For extensible classes, these queries will always be made against the most derived CodeGenerator class (that's what TR::CodeGenerator is). Thus, making them virtual provides no real value.